README-translated/README-Italian.md

```diff
@@ -24,38 +50,49 @@ Indice:
 
 ### Come contribuire
 
-:construction: Questa repository è sempre in costruzione e **tutti** sono incoraggiati ad aiutarci in questo progetto che può essere utile a molti. :construction:
+:construction: Questo repository è sempre in costruzione e **tutti** sono incoraggiati ad aiutarci in questo progetto che può essere utile a molti. :construction:
+
+Tutti sono tenuti a seguire il nostro [codice di condotta](CODE_OF_CONDUCT.md) e a leggere le nostre [linee guida](CONTRIBUTING.md) per avere più informazioni su come iniziare.
+Vogliamo rendere la contribuzione a questo progetto il più facile possibile.
+Per questo motivo, ci sono diversi modi in cui è possibile contribuire in base ai propri interessi, disponibilità e capacità.
+
+![Questa immagine mostra sei dei diversi modi in cui chiunque può contribuire. Questi sono: sviluppare e condividere, mantenere e migliorare, condividere risorse, revisione e aggiornamento, rendere il progetto globale attraverso la traduzione, e condividere le migliori pratiche.](https://github.com/alan-turing-institute/the-turing-way/blob/master/book/website/figures/README_imgs/README_contributions.png)
 
-Tutti sono tenuti a seguire il nostro [codice di condotta](CODE_OF_CONDUCT.md) e a leggere le nostre [linee guida](CONTRIBUTING.md) prima di iniziare.
+*Tra i modi di contribuire ci sono anche lo sviluppo e la condivisione di nuovi capitoli, la manutenzione e il miglioramento dei capitoli esistenti, la condivisione delle risorse de _The Turing Way_, la revisione e l'aggiornamento del materiale, la traduzione dei capitoli per rendere questo progetto accessibile globalmente, la condivisione delle miglior pratiche nell'amito della ricerca.*
 
-Se non te la senti di aiutare su GitHub, potresti anche contribuire con un caso di studio o con dei suggerimenti tramite il nostro [form su Google docs](https://goo.gl/forms/akFqZEIy2kxAjfZW2).
+Ai membri della comunità vengono date opportunità per imparare nuove abilità, condividere le proprie idee e collaborare con altri.
+Vengono anche fornite possibilità di avere, e si incoraggia poi ad essere, un mentore per contribuzioni a _The Turing Way_ e ad altri progetti open source.
+
+Abbiamo creato un [promotion pack](https://github.com/alan-turing-institute/the-turing-way/tree/master/communications/promotion-pack) per aiutarti a presentare e condividere _The Turing Way_.
 
 ### Citare _The Turing Way_
 
 Puoi citare _The Turing Way_ attraverso l'archivio del progetto Zenodo usando il DOI: [10.5281/zenodo.3233853](https://doi.org/10.5281/zenodo.3233853).
-Il DOI (digital object identifier) ci permette di archiviare le repository ed è molto utile per garantire che il lavoro venga tracciato nelle pubblicazioni accademiche.
+Il DOI (digital object identifier) ci permette di archiviare i repository ed è molto utile per garantire che il lavoro venga tracciato nelle pubblicazioni accademiche.
```

Comment:
Regarding `repository`, from [l'Accademia della Crusca](https://accademiadellacrusca.it/it/consulenza/quale-possibile-traducente-per-emrepositoryem/2859) 🧐  :

> Le attestazioni mostrano una netta prevalenza per il genere maschile invariabile (il repository/i repository, che ottengono su Google rispettivamente 93.600 e 37.800 risultati), anche se, come spesso succede per i prestiti inglesi, si registra una certa oscillazione nell’attribuzione del genere (le occorrenze per il femminile sono 8.910 per la repository e 3.750 per le repository) e del numero (le attestazioni per la forma flessa sono 3.590 per i repositories e 831 per le repositories). Generalmente il genere di un prestito viene assegnato in base al genere del nome che viene individuato o percepito come traducente: in questo caso gli equivalenti italiani più in uso sono archivio, magazzino, deposito, tutte forme maschili. L’uso del femminile non si può dunque spiegare con tale criterio: è invece il suffisso -ory (di factory, memory, ecc.) a indirizzare verso il genere femminile.

 Both would work for me, but the masculine is way more common.